### DISCUSSION TITLE
[Issue Forms] Stuck when post is too long

### DISCUSSION THREAD

**Discussion by RalfJung** [community/community#16650] [Projects and Issues]
I filled an issue form (https://github.com/k9mail/k-9/issues/new?assignees=&labels=bug&template=bug_report.yml, specifically), and the text I wote was too long so I got the error that the most is larger than 64k. However, now I was stuck: the "submit the issue" button is disabled, so even after making the text shorter, I cannot submit the issue again.

I had to open the issue again in a new tab and copy everything over, in order to submit the issue.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Reply by RalfJung** [reply]
Okay, done that: https://support.github.com/ticket/personal/0/1695005

(Sorry I had found this this here first when looking how to report a GH issue)
